signac/common/validate.py

```diff
@@ -52,7 +41,7 @@ def get_validator():
 cfg = """
 project = string()
 workspace_dir = string(default='workspace')
-signac_version = version(default='0,1,0')
+schema_version = string(default='1')
```

Comment:
Should this be a proper semantic version string?
```suggestion
schema_version = string(default='1.0.0')
```

Reply:
See comment below.